include/GafferImage/ImagePlug.h

```diff
@@ -218,6 +218,12 @@ class GAFFERIMAGE_API ImagePlug : public Gaffer::ValuePlug
 		{
 			return ( ( point.y - tileOrigin.y ) << tileSizeLog2() ) + point.x - tileOrigin.x;
 		};
+
+		/// Returns the pixel corresponding to an unwrapped index
+		inline static Imath::V2i indexPixel( int index, const Imath::V2i &tileOrigin )
+		{
+			return Imath::V2i( ( index % tileSize() ) + tileOrigin.x, ( index >> tileSizeLog2() ) + tileOrigin.y );
```

Comment:
Am I correct in thinking that we're using `>> tileSizeLog2()` to avoid an expensive division? If so, would we be better to derive `index % tileSize()` in the same way?

Reply:
Good point.  This is likely a place where we probably can trust the compiler, but I was probably the one who implemented tileSizeLog2(), and I should be consistent with that ( I guess my real rationale is that implementing it as tileSizeLog2() ensures in the future that it's always a value that allows using shifts ).